tests/core/test_model.py

```diff
@@ -233,6 +266,44 @@ async def test_fingerprinting_changed_response_text(project: Text):
     assert old_fingerprint[FINGERPRINT_NLG_KEY] != new_fingerprint[FINGERPRINT_NLG_KEY]
 
 
+async def test_fingerprinting_changing_config_epochs(project: Text):
+    importer = _project_files(project)
+
+    old_fingerprint = await model_fingerprint(importer)
+    config = await importer.get_config()
+
+    for key in ["pipeline", "policies"]:
+        for p in config[key]:
+            if "epochs" in p:
+                p["epochs"] += 10
```

Comment:
Q: Should we create test config files and pass them to this test rather than explicitly increasing the epochs inside the test? Don't have a hard opinion but just wondering what would be the best way to test it.

Reply:
Good question. The problem I see with that is someone could change the number of epochs in one of the files, which would not make this test fail, but mean it wasn't actually testing anything.

Reply:
Hmm, that makes sense. I think the place where having it in a file would be useful is when you are writing a test to check if the right number of epochs are passed to the ML components during loading. I guess that would be part of your other PR or maybe my PR.

Reply:
I think we could simplify the test by doing something like:

1. Define config as `Dict`
2. Write to a file
3. `TrainingDataImporter.load_from_config`
4. Test fingerprint